hashicorp/boundary-ui
# Redirect on reapply error

:tickets: [Jira ticket](https://hashicorp.atlassian.net/browse/ICU-12367)

## Description

This PR redirects to the session recordings list page on the `reapply` error

## Screenshots (if appropriate):



https://github.com/hashicorp/boundary-ui/assets/15043878/fcb6b728-8efb-4041-8921-3cd271492e37




## How to Test

1. get the latest ENT binary
2. create some recordings
3. go to channels view and click on manage dropdown
4. grab the ID from the page go to a terminal and delete the session recording that you are on 
5. after deletion on the CLI, clicking on `reapply` policy should redirect you to the list page and throw an error. 



:technologist: [Admin preview](PATH_TO_FEATURE)

:desktop_computer: [Desktop preview](PATH_TO_FEATURE)

## Checklist:
